save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -92,14 +92,19 @@ class Save(
         logInfo("=> Executing plugin: ${plugin::class.simpleName} for [${plugin.testConfig.location}]")
         reporter.onPluginExecutionStart(plugin)
         try {
-            plugin.execute()
+            val events = plugin.execute()
+            if (events.toList().isEmpty()) {
+                logWarn("No resources discovered for ${plugin::class.simpleName} in [${plugin.testConfig.location}], skipping")
```

Comment:
Why is it warn? I think, we consider this normal behavior, so it should be debug?

Reply:
yeah, this is my code, may be should be changed to INFO, if user has forgotten to put resources

Reply:
Changed to INFO